docs/source/guide/import_troubleshoot.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Troubleshoot import and export issues
+short: Troubleshooting
+tier: all
+type: guide
+order: 280
+order_enterprise: 180
+meta_title: Troubleshooting import and export issues in Label Studio
+meta_description: Tips for troubleshooting Label Studio issues when importing/exporting tasks and data
+section: "Import and Export"
+date: 2023-12-06 10:48:03
+---
+
+## General guidelines for importing data
+
+* For larger projects, do not upload data through the Label Studio interface. This works fine for proof of concept projects, but it is not recommended for larger projects. We recommend that you configure external storage instead. 
+* It’s best to keep about 100k tasks / 100k annotations per project for optimal performance.
+* Avoid frequent imports because each new import requires lengthy background operations. One import per 30 seconds will work without overloads.
+* If your API requests exceed our limits per request, they will be truncated. T
+
+## Troubleshoot cloud storage
+
+When working with an external Cloud Storage connection (S3, GCS, Azure), keep the following in mind:
+
+* Label Studio doesn’t import the data stored in the bucket, but instead creates *references* to the objects. Therefore, you have full access control on the data to be synced and shown on the labeling screen.
+* Sync operations with external buckets only goes one way. It either creates tasks from objects on the bucket (Source storage) or pushes annotations to the output bucket (Target storage). Changing something on the bucket side doesn’t guarantee consistency in results.
+* We recommend using a separate bucket folder for each Label Studio project.
+
+### CORS errors
+
+If you have not set up CORS, you cannot view cloud storage data from Label Studio. You might see a link to the data rather than a preview of the data, or you might see a CORS error in your web browser console:
```

Comment:
I think that we could go a bit deeper into exactly why CORS errors happen. I don't think it should hold up this pull request, but I want to add it to our backlog.